### PR TITLE
Fixes for release blocker items

### PR DESCRIPTION
**Pull request by samshara**

### CODE REVIEW

File: src/App/PageError/index.tsx [hunk 1]
```diff
@@ -63,13 +62,13 @@ function PageError() {
...
-                        to="home"
-                        variant="primary"
+                    <a
+                        className={styles.homeLink}
+                        href="/"
                     >
                         {/* FIXME: use translations */}
                         Go back to homepage
-                    </Link>
+                    </a>
```
Comment (tnagorra) [inline src/App/PageError/index.tsx]: Let's add a note why we are using anchor element instead of Link.

File: src/App/PageError/styles.module.css [hunk 2]
```diff
@@ -43,6 +43,22 @@
...
+                background-color: var(--go-ui-color-primary-red);
+                padding: var(--go-ui-spacing-4xs) var(--go-ui-spacing-sm);
+                text-decoration: none;
+                color: var(--go-ui-color-white);
+
+                &:hover {
+                    background-color: var(--go-ui-color-red-hover);
+                    text-decoration: underline;
+                }
+            }
```
Comment (tnagorra) [inline src/App/PageError/styles.module.css]: For the styling, we should be able to use useButtonFeatures hook

File: src/components/SearchMultiSelectInput/index.tsx [hunk 3]
```diff
@@ -299,7 +299,10 @@ function SearchMultiSelectInput<
             onClearButtonClick={handleClear}
             searchText={searchInputValue}
             onSearchTextChange={handleSearchValueChange}
-            dropdownShown={showDropdown}
+            dropdownShown={showDropdown && (
+                (isTruthyString(searchInputValue) && searchInputValue.length > 0)
+                || realOptions.length > 0
+            )}
```
Comment (tnagorra) [inline src/components/SearchMultiSelectInput/index.tsx]: This not not how we hide the dropdown.

We should instead not send a query to the server when we click on the dropdown.

File: src/components/SearchSelectInput/index.tsx [hunk 4]
```diff
@@ -310,7 +310,10 @@ function SearchSelectInput<
             onClearButtonClick={handleClear}
             searchText={searchInputValue}
             onSearchTextChange={handleSearchValueChange}
-            dropdownShown={showDropdown}
+            dropdownShown={showDropdown && (
+                (isTruthyString(searchInputValue) && searchInputValue.length > 0)
+                || realOptions.length > 0
+            )}
```
Comment (tnagorra) [inline src/components/SearchSelectInput/index.tsx]: This not not how we hide the dropdown.

We should instead not send a query to the server when we click on the dropdown.

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/App/PageError/index.tsx [hunk 5]
```diff
@@ -63,13 +65,14 @@ function PageError() {
...
-                    <Link
-                        to="home"
-                        variant="primary"
+                    {/* NOTE: using the anchor element as it will refresh the page */}
+                    <a
+                        className={containerClassName}
+                        href="/"
                     >
                         {/* FIXME: use translations */}
                         Go back to homepage
```
Comment (barshathakuri) [inline src/App/PageError/index.tsx:74]: Use string translations

**Review by barshathakuri — APPROVED** [review]

**Pull request merged**